pyproject.toml

```diff
@@ -138,23 +138,24 @@ search = """\
 ------------------------------------------------------------
 """
 replace = """\
-`Unreleased <https://github.com/Ouranosinc/xsdba>`_ (latest)
-------------------------------------------------------------
+..
+    `Unreleased <https://github.com/Ouranosinc/xsdba>`_ (latest)
+    ------------------------------------------------------------
 
-Contributors:
+    Contributors:
 
-Changes
-^^^^^^^
-* No change.
+    Changes
+    ^^^^^^^
+    * No change.
 
-Fixes
-^^^^^
-* No change.
+    Fixes
+    ^^^^^
+    * No change.
 
 .. _changes_{new_version}:
 
-`v{new_version} <https://github.com/Ouranosinc/xsdba/tree/{new_version}>`_
-----------------------------------------------------------
+`v{new_version} <https://github.com/Ouranosinc/xsdba/tree/{new_version}>`_ ({now:%Y-%m-%d})
+-----------------------------------------------------------------------
```

Comment:
not sure I understand, this will be changed and fixed automatically  by jinja?

Reply:
So this is where things get wild: The length of this line is generated by cookiecutter (also in jinja) dependent on the length of the URL. This will work so long as the version string stays at a fxed length (which it won't).

I might need to see what can be done on the cookiecutter side about this. 

For reference:
```
`v{new_version} <https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}/tree/{new_version}>`_
{{ '-' * ('`v' + cookiecutter.version + ' <https://github.com/' + cookiecutter.github_username + '/' + cookiecutter.project_slug + '/tree/' + cookiecutter.version + '>`_')|length }}
```